manifests/lvs/virtual_server.pp

```diff
@@ -152,7 +152,7 @@
   concat::fragment { "keepalived.conf_lvs_virtual_server_${_name}-footer":
     target  => "${::keepalived::config_dir}/keepalived.conf",
     content => "}\n",
-    order   => "250-${_name}",
+    order   => "250-${_name}-zzz",
```

Comment:
Doing that only works if you have a single group of real servers. If you have more than one, all of the groups will get created first at order 250-<name> and all the brackets later at 251. Ends up looking something like this:

> 
> group group1 {
> 	...
> 
> group group2 {
> 	...
> 
> group group3 {
> 	...
> 
> }
> }
> }

I can ponder an alternate way to build up the sections of the config file, but for the immediate timeframe this is all I've got.